source/custom-output-cli.txt

```diff
@@ -0,0 +1,250 @@
+.. _go-template-output: 
+
+=============================
+Customize the {+atlas-cli+} Output
+=============================
+
+.. default-domain:: mongodb
+
+You can customize the {+atlas-cli+} output fields and format using a ``Go`` 
+template or a JSON path.
```

Comment:
We should add a line here on why someone might want to do this. I believe this would be why, but check with engineering:
```suggestion
template or a JSON path, which makes it easier to automate processes based on the 
output from the {+atlas-cli+}.
```